news/bugfix-3141.md

```diff
@@ -0,0 +1 @@
+`secret-storage`: Fixed some cases, where log messages were truncated.
```

Comment:
For the reviewers: Is this clear, or misinterpret-able?

Reply:
I would say diagnostical logs instead of log messages, just to be sure.

Reply:
Thanks, fixed! :)